cosmos_transfer1/diffusion/datasets/augmentors/control_input.py

```
@@ -737,8 +737,16 @@ def __call__(self, data_dict) -> dict:
         if "control_input_hdmap" in data_dict:
             # already processed
             return data_dict
-        else:
-            raise ValueError("HDMAP condition is not provided.")
+        if "video" not in data_dict:
+            image = np.array(data_dict[self.input_keys[1]])
+            h, w, _ = image.shape
+            data_dict[self.output_keys[0]] = torch.from_numpy(np.zeros((3, h, w)).astype(np.uint8))
```

Should we raise an exception since we don't currently support image input?

---

I'm leaning towards removing this segment completely as data_dict["video"] shouldn't have any interaction with the hdmap augmentor - currently this is not checking whether the video for the hdmap is in data_dict, but rather whether there's a rgb video in data_dict.